SalesforceCommerceCloud/pwa-kit
# Fix HTML Mismatch Warning

## Issue

When rendering pages like the home page or product list page, you'll see console errors complaining about mismatch classname in the html. This is the result of a few components conditionally rendering either an `Img` component or an `Image` component based on whether you are on the server or client. 

This pattern was used to hopefully take advantage of loading skeletons, but had been more troublesome than not. So for the time being it was recommended that we simply select one type of component to use and stick with it. We'll then open up ourselves to open source submissions or have it works its way on to our sprint board.

 **JIRA**: [W-9126009](https://gus.lightning.force.com/lightning/_classic/%2Fa07AH000000AekEYAS)
 **Linked PRs**: None

## How to test-drive this PR
- Checkout `develop` branch and start the server.
- Goto `http://localhost:300` and notice that you will receive a warning about mismatch classname in the html.
- No checkout _this_ branch and do the same
- The error should not occur. (NOTE: This also happens on the product list page)

## Changes
- Remove conditional usage of `Image` component.
